lte/gateway/c/core/oai/tasks/nas/esm/EpsBearerContextDeactivation.c

```diff
@@ -417,8 +417,8 @@ status_code_e eps_bearer_deactivate_t3495_handler(
   ue_mm_context_t* ue_mm_context = mme_app_get_ue_context_for_timer(
       ue_id, "EPS BEARER DEACTIVATE T3495 Timer");
   if (ue_mm_context == NULL) {
-    OAILOG_ERROR(
-        LOG_MME_APP,
+    OAILOG_ERROR_UE(
+        LOG_MME_APP, ue_mm_context->emm_context._imsi64,
```

Comment:
ue_mm_context does not exist in this case, let's not change this

Reply:
Resolved

Reply:
Resolved